src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -0,0 +1,112 @@
+import _ from 'lodash';
+
+import { getConnection } from 'api/v1/db/oracledb/connection';
+import { getDoughById } from 'api/v1/db/oracledb/doughs-dao';
+import { serializePizza } from 'api/v1/serializers/pizzas-serializer';
+
+const pizzaColumns = {
+  id: 'ID',
+  doughId: 'DOUGH_ID',
+  name: 'NAME',
+  bakeTime: 'BAKE_TIME',
+  ovenTemp: 'OVEN_TEMP',
+  specialInstructions: 'SPECIAL_INSTRUCTIONS',
+};
+
+/**
+ * A list of columns in the PIZZAS table aliased to their
+ * corresponding PizzaResource attribute names
+ *
+ * @const {string}
+ */
+const pizzaColumnAliases = _.map(pizzaColumns,
+  (columnName, attributeName) => `${columnName} AS "${attributeName}"`)
+  .join(', ');
+
+/**
+ * A SQL query to get a pizza with a specific ID
+ *
+ * @const {string}
+ */
+const getPizzaByIdQuery = `SELECT ${pizzaColumnAliases} FROM PIZZAS WHERE ID = :id`;
+
+/**
+ * Fetch all the ingredients associated with the pizza with id
+ *  `pizzaId`
+ *
+ * @param {string} pizzaId
+ * @returns {Array} an array of unserialized ingredient objects
+ */
+const getPizzaIngredients = async (pizzaId) => {
+  const query = `SELECT
+    INGREDIENTS.ID AS "id",
+      INGREDIENTS.TYPE AS "ingredientType",
+      INGREDIENTS.NAME AS "name",
+      INGREDIENTS.NOTES AS "notes"
```

Comment:
Just a little nitpicking, I think the query here is not aligned and line breaking points are a little strange to me. It's not a big deal here, but it makes your query a bit hard to read if the query becomes larger, e.g. `getPizzaByIdQuery`. Take a look at [this file](https://github.sig.oregonstate.edu/ecs-data/students-api-contrib/blob/master/contrib.js) and if the way we organized queries can help a bit.

Reply:
I think I got it into a format that looks like what's used in the other file, let me know what you think

Reply:
Much better. Make you modify other queries (include queries in `test-data.js`) as well.

Reply:
Although a lot of the other ones are more procedurally generated and would be hard to split up. Was looking over the other files and they seem like they're in OK shape style wise too.